pkg/cloud/aws/sqs/service.go

```diff
@@ -201,7 +201,7 @@ func (s *service) GetUrl(ctx context.Context, name string) (string, error) {
 			return "", nil
 		}
 
-		return "", nil
+		return "", err
```

Comment:
Do you mind adding an error wrap here? To add some context to the error?  
e.g. `fmt.Errorf("can not request queue url: %w", err)`  
I know it seems redundant from time to time, but out of experience it's better to have one information to much than to less..

Reply:
sure